lib/DependencyScan/ModuleDependencyCacheSerialization.cpp

```diff
@@ -422,10 +409,14 @@ bool ModuleDependenciesCacheDeserializer::readGraph(
       if (!bufferIdentifier)
         llvm::report_fatal_error(
```

Comment:
report_fatal_errors are bad in here. Some of them are going in this PR but there are more left.

Reply:
Lots of these messages in this code in general, something to look into in a different change later.